docker-compose.yml

```
@@ -198,6 +197,142 @@ services:
+      APP_PROVIDER_WOPI_INSECURE: true
+      APP_PROVIDER_WOPI_WOPI_SERVER_EXTERNAL_URL: https://${WOPISERVER_DOMAIN:-host.docker.internal:8880}
+      APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL: https://${OCIS_DOMAIN:-host.docker.internal:9200}
+      # share the registry with the ocis container
+      MICRO_REGISTRY: 'mdns'
+    volumes:
+      - ocis-config:/etc/ocis
+    extra_hosts:
+      - host.docker.internal:${DOCKER_HOST:-host-gateway}
+    restart: always
+    depends_on:
+      - ocis
+      - collabora
+      - wopiserver
+
+  ocis-appprovider-onlyoffice:
+    image: ${OCIS_IMAGE:-owncloud/ocis:latest}
+    user: '0' # needed for apk add in entrypoint script
+    entrypoint:
+      - /bin/sh
+      - /entrypoint-override.sh
+    #command: app-provider server
+    environment:
+      APP_PROVIDER_LOG_LEVEL: debug
+      # use the internal service name of the gateway
+      REVA_GATEWAY: ${REVA_GATEWAY:-com.owncloud.api.gateway}
+      APP_PROVIDER_GRPC_ADDR: 0.0.0.0:9164
+      # configure the service name to avoid collision with collabora
+      APP_PROVIDER_SERVICE_NAME: app-provider-onlyoffice
+      # use the internal service name
+      APP_PROVIDER_EXTERNAL_ADDR: com.owncloud.api.app-provider-onlyoffice
+      APP_PROVIDER_DRIVER: wopi
+      APP_PROVIDER_WOPI_APP_NAME: OnlyOffice
+      APP_PROVIDER_WOPI_APP_ICON_URI: https://${ONLYOFFICE_DOMAIN:-host.docker.internal:9981}/web-apps/apps/documenteditor/main/resources/img/favicon.ico
+      APP_PROVIDER_WOPI_APP_URL: https://${ONLYOFFICE_DOMAIN:-host.docker.internal:9981}
+      APP_PROVIDER_WOPI_INSECURE: true
+      APP_PROVIDER_WOPI_WOPI_SERVER_EXTERNAL_URL: https://${WOPISERVER_DOMAIN:-host.docker.internal:8880}
+      APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL: https://${OCIS_DOMAIN:-host.docker.internal:9200}
+      # share the registry with the ocis container
+      MICRO_REGISTRY: 'mdns'
+    volumes:
+      - ./dev/docker/ocis-appprovider-onlyoffice/entrypoint-override.sh:/entrypoint-override.sh
+      - ocis-config:/etc/ocis
+    extra_hosts:
+      - host.docker.internal:${DOCKER_HOST:-host-gateway}
+    restart: always
+    depends_on:
+      - ocis
+      - onlyoffice
+      - wopiserver
+
+  collabora:
+    image: collabora/code:22.05.10.1.1
+    environment:
+      DONT_GEN_SSL_CERT: 'YES'
+      extra_params: --o:ssl.enable=false --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=https://host.docker.internal:9200
+    cap_add:
+      - MKNOD
+    extra_hosts:
+      - host.docker.internal:${DOCKER_HOST:-host-gateway}
+    labels:
+      traefik.enable: true
+      traefik.http.routers.collabora.tls: true
+      traefik.http.routers.collabora.rule: PathPrefix(`/`)
+      traefik.http.routers.collabora.entrypoints: collabora
+      traefik.http.services.collabora.loadbalancer.server.port: 9980
+    restart: always
+    depends_on:
+      - traefik
+
+  onlyoffice:
+    image: onlyoffice/documentserver:7.3.0
```

```suggestion
    image: onlyoffice/documentserver:7.3.3
```
Recently a version of Openoffice is updated in Ocis. Can we use the same version or not?

---

@SagarGi aren't we going to use updated version?